extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -24,6 +35,10 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 	function createProvider(logGroup, providerName, src, slotMap, extra) {
 		extra = extra || {};
 
+		if (extra.overrideSizesPerCountry && extra.overrideSizesPerCountry[country]) {
+			overrideSizes(slotMap, extra.overrideSizesPerCountry[country]);
```

Comment:
Another nitpicking nAndy: how about moving the `overrideSizes()` method to GPT helper? This way we won't introduce two more dependencies here (just one more - `geo` in the helper) and it feels more like this kind of function isn't a part of factory. But I might be missing a point and you don't have to agree. Just tell me what do you think about it :japanese_ogre: (since creating helpers and utils is also quite a bad habit to some degree :D)